tests-e2e/test1/entry_point.rs

```diff
@@ -9,7 +9,22 @@ pub struct Point {
     y: i32,
 }
 
+#[wasm_bindgen]
+pub fn consume(point: Point) {}
+
 #[wasm_bindgen]
 pub fn into_js() -> Point {
     Point { x: 0, y: 0 }
 }
+
+#[wasm_bindgen]
+pub fn consume_vector(points: Vec<Point>) {}
+
+#[wasm_bindgen]
+pub fn vector_into_js() -> Vec<Point> {
+    vec![
+        Point { x: 1, y: 6 },
+        Point { x: 2, y: 5 },
+        Point { x: 3, y: 4 },
+    ]
+}
```

Comment:
This test looks great, but could you please create a new test folder so the previous test remain unchanged?

Reply:
Done